libr/util/print.c

```diff
@@ -733,7 +733,7 @@ static bool isAllZeros(const ut8 *buf, int len) {
 }
 
 #define Pal(x,y) (x->cons && x->cons->context->pal.y)? x->cons->context->pal.y
-R_API void r_print_hexii(RPrint *rp, ut64 addr, const ut8 *buf, int len, int step) {
+R_API void r_print_hexii(RPrint *rp, ut64 addr, const ut8 *buf, int len, int step, int show_offset) {
```

Comment:
this is bool too

Reply:
but.. what about adding a field inside RPrint->show_offset instead of adding a parameter?

Reply:
Yeah that sounds better, fixing it